uiowa/uiowa
# Feature: Block content templates in uids_base

This resolves #1740. The issue is that a block placed outside of the layout builder doesn't pick up our block templates where we have implemented UIDS components. This PR adds a module that causes block content to get its own preprocess function and template.

# How to test
- `composer install`
- Add Card to a block region like `pre-footer`. 
- Inspect the HTML of the card and verify that it is picking up the `block-content--uiowa-card.html.twig` template.
- Verify that markup on uiowa.local.drupal.uiowa.edu for blocks placed by Layout Builder is not broken.
